include/drivers/sensor_sim.h

```diff
@@ -21,6 +21,13 @@ extern "C" {
 #include <drivers/sensor.h>
 #include <wave_gen.h>
 
+/**
+ * @defgroup sensor_sim Simulated sensor driver
+ * @{
+ * @brief Simulated sensor device driver.
+ *
+ */
+
 /** @brief Set simulated acceleration parameters.
  *
  * @note	This function can be used only if acceleration is generated as wave signal.
```

Comment:
Don't use @warning in line 34. This is a very critical notification type that should be used very rarely. Merge the two notes, I'd say:
```suggestion
 * @note	This function can be used only if acceleration is generated as wave signal.
 *              Moreover, although it is thread-safe, it cannot be used in interrupts.
```